modules/oc-mirror-mirror-to-disk.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-mirroring-disconnected.adoc
+
+[id="oc-mirror-mirror-to-disk_{context}"]
+= Mirroring from a mirror to disk
+
+// TODO: intro
+
+// TODO: usage.md mentioned that this might download hundreds of gigs. And take awhile. Add this warning? And that you can add "log-level=debug" to track progress
```

Comment:
@afflom @jpower432 Do we need a warning about the potential download size and/or how long it might take to complete?